README.rst

```diff
@@ -1,32 +1,28 @@
-+----------------------+----------------------------------------------------------------+
-| master branch status | |BuildStatus| |Docs| |MITLicense| |CoverageStatus|             |
-+----------------------+----------------------------------------------------------------+
-| GitHub status        | |OpenIssues| |OpenPullRequests|                                |
-+----------------------+------------+---------------------------------------------------+
-| Versions             | NI-DCPower | |nidcpowerLatestVersion| |nidcpowerPythonVersion| |
-|                      +------------+---------------------------------------------------+
-|                      | NI-DMM     | |nidmmLatestVersion| |nidmmPythonVersion|         |
-|                      +------------+---------------------------------------------------+
-|                      | NI-FGEN    | |nifgenLatestVersion| |nifgenPythonVersion|       |
-|                      +------------+---------------------------------------------------+
-|                      | NI-SCOPE   | |niscopeLatestVersion| |niscopePythonVersion|     |
-|                      +------------+---------------------------------------------------+
-|                      | NI-SWITCH  | |niswitchLatestVersion| |niswitchPythonVersion|   |
-|                      +------------+---------------------------------------------------+
-|                      | NI-ModInst | |nimodinstLatestVersion| |nimodinstPythonVersion| |
-+----------------------+------------+---------------------------------------------------+
++----------------------+---------------------------------------------------------------------------------------------+
+| master branch status | |BuildStatus| |Docs| |MITLicense| |CoverageStatus|                                          |
++----------------------+---------------------------------------------------------------------------------------------+
+| GitHub status        | |OpenIssues| |OpenPullRequests|                                                             |
++----------------------+------------+--------------------+----------------------------+------------------------------+
+|                      | **Name**   | **Driver Version** | **PyPI Version**           | **Supported Python Version** |
+|                      +------------+--------------------+----------------------------+------------------------------+
+| Versions             | NI-DCPower | 17.6.0             | |nidcpowerLatestVersion|   | |nidcpowerPythonVersion|     |
+|                      +------------+--------------------+----------------------------+------------------------------+
```

Comment:
Why do we hard code values directly on the table, but for the other columns we use variables?
I'm ok hardcoding the values directly on the table as long as we're consistent and we aren't repeating ourselves (variables may be overkill if they are only used once).

Reply:
The variable I believe are there to enable the little shield things. Such as https://img.shields.io/pypi/v/nidcpower.svg . We don't really have such a thing for the NI Drivers.

I plan to use the hardcoded values unless you have any concerns?